### PR TITLE
Add option to disable translation button

### PR DESCRIPTION
**Pull request by quadratz**
Hopefully, this fix addresses the over-indexing by the Algolia crawler.
For reference: https://t.me/c/1322374388/5994

### CODE REVIEW

File: site/docs/.vitepress/config.ts [hunk 1]
```diff
@@ -50,5 +51,21 @@ export default defineConfig({
     build: {
       chunkSizeWarningLimit: 1600,
     },
+    resolve: {
+      alias: [
+        {
+          find: /^.*\/VPNavBar\.vue$/,
+          replacement: fileURLToPath(
+            new URL("./components/CustomNavBar.vue", import.meta.url),
```
Comment (KnorpelSenf) [inline site/docs/.vitepress/config.ts:59]: Isn't there `import.meta.dirname`?

**Comment by KnorpelSenf** [comment]
This is very, very cool stuff

**Comment by KnorpelSenf** [comment]
#1014 will fix CI

File: site/docs/.vitepress/config.ts (hunk 1 above)
Comment (quadratz) [reply site/docs/.vitepress/config.ts]: That works as well. I've set a minimum required version of Node.js as a safeguard, considering it's a new API. (https://github.com/grammyjs/website/pull/1011/commits/ab639136e59193762dbfb699718e8f54ceb89e0c)

**Review by roj1512 — APPROVED** [review]

**Pull request merged**